src/daemon/rpc_command_executor.cpp

```diff
@@ -2398,32 +2398,42 @@ static void append_printable_service_node_list_entry(cryptonote::network_type ne
     buffer.append(indent1);
     buffer.append("[");
     buffer.append(std::to_string(entry_index));
-    buffer.append("] Service Node: ");
+    buffer.append("] ");
+    buffer.append("Service Node: ");
```

Comment:
This is mainly just style, and I know this is just following the existing code, but all of these `buffer.append(blah);` would look so much nicer as `buffer += blah;`

Reply:
... or `std::ostringstream o; o << blah; ....; buffer += o.str();` which would also save a bunch of `std::to_string`s.

Reply:
Added